submissions/HannaSyn/Popup/index.html

```diff
@@ -0,0 +1,87 @@
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/blogs.png" alt="blogs">
+                <p>Blogs</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/chat.png" alt="chat">
+                <p>Chat</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/calendar.png" alt="calendar">
+                <p>Calendar</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/files.png" alt="files">
+                <p>Files</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/photos.png" alt="photos">
+                <p>Photos</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/videos.png" alt="videos">
+                <p>Videos</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/documents.png" alt="documents">
+                <p>Documents</p>
+              </a>
+              <a href="#" class="menu-popup__link"><img src="assets/popup-icons/contacts.png" alt="contacts">
+                <p>Contacts</p>
+              </a>
+                <input class="menu-popup__input" type="checkbox" id="more-btn" />
+                <label class="menu-popup__label" for="more-btn"></label>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/maps.png" alt="maps">
+                <p>Maps</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/search.png" alt="search">
+                <p>Search</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/classroom.png" alt="classroom">
+                <p>Classroom</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/mars.png" alt="mars">
+                <p>Mars</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/promo.png" alt="promo">
+                <p>Promo</p>
+              </a>
+              <a href="#" class="menu-popup__link nondisplay"><img src="assets/popup-icons/tanslation.png" alt="translation">
+                <p>Translation</p>
+              </a>
+            </div>
+          </div>
+        </li>
+        <li class="navigation__link">
+          <a  href="#"><img src="assets/bell.svg"
+              alt="bell"></a>
+        <li  class="navigation__link link">
+          <a  href="https://github.com/HannaSyn/kottans-frontend" class="link__user"><img src="assets/user.svg"
+              alt="user"></a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main">
+    <figure class="bg">
+      <img class="bg__img" src="assets/bg.png" alt="">
```

Comment:
[Read more about `object-fit`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) to avoid a situation like this:
![image](https://user-images.githubusercontent.com/16196199/96768709-d1e84c80-13e6-11eb-8080-8f236eb10bf6.png)

Reply:
I thought that responsive is unnecessary in this project.

Reply:
It's just a recommendation, the main point of the task is popup, if I see some issues like this I just give advice on how to avoid it. Fix it or not is up to you.)